AUTHORS

```diff
@@ -693,6 +693,7 @@ answer newbie questions, and generally made Django that much better:
     Mattia Procopio <promat85@gmail.com>
     Mattias Loverot <mattias@stubin.se>
     mattycakes@gmail.com
+    Mavrick Laakso <laakso.mavrick@gmail.com>
```

Comment:
Unfortunately significant changes to the codebase are required to request adding oneself to the authors list, sorry 🤷‍♂️

Reply:
Understood. I will remove myself.